### ISSUE TITLE
Troubleshooting steps?

### ISSUE THREAD

**Issue by ImDevinC** [bluesky-social/pds#143]
Is there any documentation on troubleshooting steps to try and identify if an issue is with my PDS or with the greater Bsky ecosphere? In my current situation, I can sign in to my account hosted on my person PDS from my laptop, but not from my mobile device. Unfortunately the mobile device doesn't give me any logs to triage with, and the PDS doesn't show anything either.

**Comment by ImDevinC** [comment]
This was a dumb issue on my side regarding an IP address change, but the overall question still stands of good ways to troubleshoot

**Comment by sparksis** [comment]
it would be a good idea for this repo to have the wiki enabled such that users can document the the caveats.

**Comment by axfelix** [comment]
Agreed -- I think we'll try to move toward a knowledge base model in the near future :)

Closing for now, but stay tuned.
